pkg/nodeinit/calico-init_linux.go

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) 2021 Tigera, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package nodeinit contains the calico-node -init command, which is intended to be run from
+// an init container to do privileged pre-flight initialisation.  At present, it mounts
+// the BPF filesystem so it is only useful for BPF mode.
+package nodeinit
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strings"
+	"syscall"
+
+	"github.com/projectcalico/node/pkg/startup"
+	"github.com/sirupsen/logrus"
+)
+
+const bpfFSMountPoint = "/sys/fs/bpf"
+
+func Run() {
+	// Check $CALICO_STARTUP_LOGLEVEL to capture early log statements
+	startup.ConfigureLogging()
+
+	err := ensureBPFFilesystem()
+	if err != nil {
+		logrus.WithError(err).Error("Failed to mount BPF filesystem.")
+		os.Exit(2)
```

Comment:
Does 2 have a special meaning here?  If so, maybe worth a comment to explain that.